build.gradle.kts

```diff
@@ -228,18 +228,17 @@ tasks {
     }
 
     listProductsReleases {
-        sinceBuild.set("232.*")
-        System.getenv("IDE")?.let {
-            types.set(listOf(it))
-        } ?: types.set(listOf("IU", "RD", "PY"))
-
+        val ides = System.getenv("IDE")
+        types.set(ides?.split(',') ?: listOf("IU", "RD", "PY"))
+        sinceBuild.set(if (types.get() == listOf("IU")) "222.*" else "232.*")
```

Comment:
tbh we would want newer versions and the old version, would this do that?

Reply:
we want IU 222 correct? the cornjob sets the env with IDE=IU, and when thats the case we do 222.*

Reply:
ps, on running `IDE=IU ./gradlew listProductsReleases`

```
IU-2023.2.2
IU-2023.1.5
IU-2022.3.3
IU-2022.2.5
```